Userland/Libraries/LibJS/Tests/builtins/Symbol/Symbol.js

```diff
@@ -17,3 +17,11 @@ test("constructing symbol from symbol is an error", () => {
         Symbol(Symbol("foo"));
     }).toThrowWithMessage(TypeError, "Cannot convert symbol to string");
 });
+
+test("setting new properties on a symbol is an error in strict mode", () => {
+    expect(() => {
+        "use strict";
+        var symbol = Symbol("foo");
+        symbol.bar = 42;
+    }).toThrow(TypeError);
```

Comment:
don't use plain toThrow, use the one that also checks the message. JS has type errors all over the place, so this test is too vague.